### ISSUE TITLE
Fails when it tries to create WIN10 environment -- I think.

### ISSUE THREAD

**Issue by abelgaxiola** [pluralsight/PS-AutoLab-Env#130]
This is the error I get when I execute either Unattend-Lab or Setup-Lab:

> GetResourceDownload : A parameter cannot be found that matches parameter name 'ThrowOnError'.
> At line:114 char:41
>          $resource = GetResourceDownload @PSBoundParameters;
>                                          ~~~~~~~~~~~~~~~~~~
>     CategoryInfo          : InvalidArgument: (:) [GetResourceDownload], ParameterBindingException
>     FullyQualifiedErrorId : NamedParameterNotFound,GetResourceDownload

> Resource download 'E:\AutoLab\ISOs\WIN10_x64_ENT_RS2_EN_Eval.iso' checksum does not match 'CF812813211ADD8FE7C53B07B7CAFF8C'.
> At line:135 char:13
>             throw ($localized.ResourceChecksumMismatchError -f $Desti ...
>             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>      CategoryInfo          : OperationStopped: (Resource downlo...53B07B7CAFF8C'.:String) [],  RuntimeException
>     FullyQualifiedErrorId : Resource download 'E:\AutoLab\ISOs\WIN10_x64_ENT_RS2_EN_Eval.iso'  checksum does not match 'CF812813211ADD8FE7C53B07B7CAFF8C'.

Anyone having a similar issue?

My environment: Windows 10 Enterprise, Version 10.0.14393, 32 GB RAM, System Type x64

**Comment by ronpaul2020** [comment]
I've tried this on two different computers now. Both are receiving the same error you posted.

**Comment by ronpaul2020** [comment]
This is what I did in order to get this to run. After seeing your issue on two separate systems. I tried on a third system. Fresh install of win 10 fully patched. Although this time I used version 3.0.0 of autolab instead of the current version 3.0.2. It ran pretty much flawlessly, the test afterwards took 9 minutes. So I tried to duplicate the same thing on one of the systems that was having issues like yours. I'm totally new to this so there is a very good chance what I did is not best practices. Although it ended up working.   
in PowerShell from c:\auotlab\configurations\windows10 ran: wipe-lab
then ran get-module x* | uninstall-module -force
Deleted c:\PS-Autolab-Env-Dev 
Deleted c:\autolab
rebooted
downloaded version 3.0.0 (instead of the current version 3.0.2) 
unpacked to c:
Launched Powershell with admin, cd c:\ps-autolab-env-3.0.0 
ran setup-host.ps1
lots of errors about liability modules cert issues. 
This is where it gets sketchy. 
Went here https://github.com/VirtualEngine/Lability downloaed. Copied contents of file. 
Went here C:\Program Files\WindowsPowerShell\Modules\Lability\0.11.0 deleted the contents. Then pasted the files from the download. 
re ran setup-host, rebooted.
Then go to https://github.com/jdhitsolutions/Windows10 and click the Download button. Extract the CONTENTS of the Windows10-Master folder to \AutoLab\Configurations\Windows10.
 ran unnatend-lab from c:\autolab\configurations\windows10
Everything was smooth. Had to re run the test a few times. Test took 30 minutes. 
One thing that I noticed is instead of downloading the ISO Win10_x64_ENT_RS2, it downloaded an older version which was named RS1. That passed the chesksum with no issue. Hopefully this is helpful and doesn't catch anything on fire.
